frigate/api/defs/events_body.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+from typing import Optional, Union
+
+from pydantic import BaseModel, Field
+
+
+class EventsSubLabelBody(BaseModel):
+    subLabel: str = Field(title="Sub label", max_length=100)
+    subLabelScore: Optional[float] = Field(
+        title="Score for sub label", default=None, gt=0.0, le=1.0
```

Comment:
The `gt=0.0` will not allow a subLabelScore of 0.
 
From the old code the `subLabelScore` could be 0

```python
  if new_score is not None and (new_score > 1.0 or new_score < 0):
        return JSONResponse(
            content=(
                {
                    "success": False,
                    "message": new_score
                    + " does not fit within the expected bounds 0 <= score <= 1.0",
                }
            ),
            status_code=400,
        )
```

Did it make sense for it to be 0?